Convert to the expected pointer.  This was not fixed before the file
was removed upstream.

diff --git a/modules/other/gail/gailtreeview.c b/modules/other/gail/gailtreeview.c
index 23923c59a16a6efb..ac6ff0a1c4a2fe22 100644
--- a/modules/other/gail/gailtreeview.c
+++ b/modules/other/gail/gailtreeview.c
@@ -995,7 +995,7 @@ gail_tree_view_ref_child (AtkObject *obj,
         gail_cell_add_state (cell, ATK_STATE_FOCUSABLE, FALSE);
         if (focus_index == i)
           {
-            gailview->focus_cell = g_object_ref (cell);
+            gailview->focus_cell = ATK_OBJECT (g_object_ref (cell));
             gail_cell_add_state (cell, ATK_STATE_FOCUSED, FALSE);
             g_signal_emit_by_name (gailview,
                                    "active-descendant-changed",
